### ISSUE TITLE
black and white color is not replicating to transparent color tab

### ISSUE THREAD

**Issue by Akshay0701** [mrudultora/Colorpicker#10]
**Describe the bug**
black and white color is not replicating to transparent color tab 

**To Reproduce**
1 open pop up dialog 
2 change color to black color 

**Expected behavior**
the transparent color bar should be black 

**Screenshots**
![20210618_190813](https://user-images.githubusercontent.com/52353967/122570143-1a850200-d069-11eb-80d0-02a7e0e50109.gif)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mrudultora** [comment]
Yeah, that's a valid issue. I would fix that. In the time being, you can get rid of it by the steps below.
• Once you set the color to black, move the cursor in the middle bar (hue).
• You will notice that the color of transparent bar has changed to black.
I would fix it soon.

**Comment by Akshay0701** [comment]
okay, Nice hack bhai
